ctpolicy/ctpolicy.go

```diff
@@ -42,14 +44,36 @@ func New(pub pubpb.PublisherClient, sctLogs loglist.List, infoLogs loglist.List,
 	)
 	stats.MustRegister(winnerCounter)
 
+	operatorGroupsGauge := prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "ct_operator_group_size_gauge",
+			Help: "Gauge for CT operators group size, by operator and log source (capable of providing SCT, informational logs, logs we submit final certs to).",
+		},
+		[]string{"operator", "source"},
+	)
+	stats.MustRegister(operatorGroupsGauge)
+
+	shardExpiryGauge := prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "ct_shard_expiration_seconds",
+			Help: "CT shard end_exclusive field expressed as Unix epoch time, by operator and logID.",
+		},
+		[]string{"operator", "logID"},
+	)
+	stats.MustRegister(shardExpiryGauge)
+
+	exposeLoglistMetrics(sctLogs, infoLogs, finalLogs, operatorGroupsGauge, shardExpiryGauge)
+
 	return &CTPolicy{
-		pub:           pub,
-		sctLogs:       sctLogs,
-		infoLogs:      infoLogs,
-		finalLogs:     finalLogs,
-		stagger:       stagger,
-		log:           log,
-		winnerCounter: winnerCounter,
+		pub:                 pub,
+		sctLogs:             sctLogs,
+		infoLogs:            infoLogs,
+		finalLogs:           finalLogs,
+		stagger:             stagger,
+		log:                 log,
+		winnerCounter:       winnerCounter,
+		operatorGroupsGauge: operatorGroupsGauge,
+		shardExpiryGauge:    shardExpiryGauge,
```

Comment:
I don't believe these two fields are ever actually used outside of this function. If that's the case, we don't need them on the struct at all.

Reply:
You're right, but I need to figure out how to access the metrics in ctpolicy_test.go.

Reply:
Oh, I think I'm actually fine with these being on the struct for the sake of testing ☺️.